apphub/image_classification/pyramidnet/pyramidnet.ipynb

```diff
@@ -0,0 +1,327 @@
3IVbnvFdr9NTfsasZv3yM71+Vm17p5+7Wmjx6zk6Om7zA0d8h+VMtd3m5ekhm3dv7LH5+i2bbR7zvoWzpd3Pq92ya4fNjz399LxsYHTKb7O0yeZdG/0xO374qM0HT/hZuD3r/fqXu6ceHfU3JDqsr3jeWdwZAMsWZ6AAAGRAAQUAIAMKKAAAGTQ+iQgAsOKEELZIcqPR3hJjvPNc789yQgEFgAtbQdJ2k/v5pPgXDRfQyYlJm09P+47PXMmvemrSd89OTfjt5lv9p8zHB2o2n5nyy+db/GzeXGKG7ZrNW23es8V3ps4kHsD2XS+0+WUvfqXNv3X7l+dlF/kRtgrNfn7w1JTv2h3q9x3Qq9b4ecDV6J/b5S6W/WukVl2ZjxfA2cF3oAAAZEABBQAgAwooAAAZUEABAMiAAgoAQAYNd+GGRCeo8iUbT8342lyR74aNubbE8n6G7cSkn2danfU72r5utc1LLb4LN5/3h2bd1m02Hxv2+1Nr9TN1K4W8zV/6lp+cl40eP2CXfeKB+2ze1tVh8+EBP/e3becqm48M+rnCX/rwR23e3e3Xs67bdwtfesUOm5ea/Gskn/fPbUl+HnOs+vnK5bw/9gp04V5IQggtkq6StEvSekltkmYlDUk6Kum+GGPvWdhuXtJOSZfNbbdTUqukGUkjkg5IeizG6P/Dx5LB70ABXDBCCF2S3iHpTZKukxLv6L+z/AFJN0v6eIzRv2M9/TZzkl4i6dWSXiXphZKaGrjfEUmflvTnWbc9t5798r/zPJ2vhOSZk/XeGON7Mmxn2eIjXAArXgihLYTw3yQdkvQBSS/VaYrnnO2Sfl7SvSGEr4YQXrWAbT4nhPB7ko5Iuk3Sr0l6mRoonnM2S/oPc9v+bAhhZ6PbxrlBAQWwooUQrpX0sKRfVP1j2qxeLumWEEKjZ3P/SdJ/lLRhEdv8ttdL+lYI4Y1nYF04QyigAFasEMLrJX1N0o4zudozuK6FaJd0cwjhh87T9nESvgMFsCKFEF4h6e8l+U5HaVTSF+b+7Zd0XPWTinWSNqn+neWNOjNnkN82K+kBSXdJukfSMUkDksZUL5A9kq6WdIOk79X8k5y8pL8MITwcY3y8wW1+cm69Ke2SXFH+v5IW0kT1zQUsuyI0XEBLbf41GCp++ZlJ/yatKj/Dtq3gu2EnE12102W/3XzRr6e9q8vmLe2+yzcmTs6nExtuau/0ecfaxHZ9p+zE1PxO2eFRv821W/1XIs1t/lOq8VE/h3igt8/m1bLfbq3mn8PpGb/8+KTvbi1P+/nBzU1+//O5xN/B4J/DWvRduPWejvlizj8uLD8hhI2qFw73opmV9HuSfj3GOHaK1Xws1Lto3ijpNyVdnnF3oqRbJf21pJtjjIOnWf5Lkn43hLBH0kckfc9Jt7dI+l1Jb2ho4zH+v6e6PYSwQ76A/laM8dZGtnGh4iNcACvR/5E/6xqXdEOM8d2nKZ6SpFj3adXPCt8nJc4AvKqkT0i6Osb4qhjjnzRQPJ+97SdUPwu+x9z8+hDCZQvYF5wFfIQLYEUJIbxE0k3mpoqk18YY3bUvTynGWJH0qyGEuyVNN3i3n4sx+stYNb7dqRDCv5PkfsbyRkl7F7N+LA5noABWml9I5L+bpXg+W4zxs40OV1hs8XzWeu6XdK+56fozsX5kRwEFsGLMfff5WnPTCUnvPce7cya5wn/JOd8LfBcKKICV5A2qd6qe7GMxRn+l+eXhsMk2nvO9wHdp+DvQQrOfHxrHfZdsLtFJWYuJrw+KqV3xXbVq8v8tFFId6zm//x09qe5u3yvw1MP+K4dVifWsXr/H5ht3+Xy0/+i8bHzwuF12/dpdNj9x6BmbT437LtypCd8NOz7q5/sWi362bXnWd+FWpv2xLI/57t9cyX/ylS8nntuqH+xSKydaxLGS3ZDI/+Kc7sWZ5wZZN4UQWs/UR8VYOJqIAKwkzzfZuKSHzvWOpMz9NGaXpOfN/d/OuX/tSn8qeFEib5NEAT1PKKAAVoQQQkn1gnSyB2KM5/VHvnMD5V8l6cck/YDqBfNMSA2JwDlAAQWwUmySH7P35LnekWcLIXyPpD+WdOVZWH3i2nw4F2giArBSdCdyf2HbcyCE8F5Jt+vsFE+Jv+HnVcNnoNWqH8c2nhgzV6n6TxZizi8/M+WbgqYSn1DMJvanmvPj22qJ5ZtaEs1OVb+fE6P+64aYGG/38J2327x/oN/mXWvmj/5b1bPJLtvU5hu4Thw5ZvPhQT8EJZfz/w2mjk2u6J+Tai1x7Cv+gtdTQ745arLq96fd9y4pVH1jWi7xnCdH+Z3fT/mweIlXiHw33FkWQvh1Sb9yPraNc4N3LwBWitRVUs75O6MQwg06dfH8purXB329pN2qnz0XY4zB/ZP042d/r7FQfAcKYKVIjdg7Uw07DQkh5CX9r8TNj0n6DzHGLy9wtY1c/BvnGAUUwErhfispSavO6V5Ir5D/zvN+1QfZp/bzVM71Y0AD+AgXwEoxfwpJnb/u39njLjNWkfT2jMVTklYvYn9wllBAAawIc6P63Mg7N1zhbHqpyb4SY3x0Eeu8dBH3xVnS+Ee4iQtbl6LvdGxr8XkuccHrqbL/+mJwyHfDxrz/RCPm/FcFlenEGLt+3wnanGj53LZjm81b2/3+HD4yYPMT+5+yeW16/iUKi4nWiL3HDti8POWP/aZd/nrA04kO6KbhYZuP9PvHVCv77cYOfyx3XfMim1+5x49FXNXme0Fma4mrqye7i/3FzGemTnt5SCx990vaclK2OoRwaYzxsXO0DxtMdtsi1/mCRd7/VGIiTzVlYQ5noABWklRzzo+ew32Y/1u0+tVgMgkhXCLJv3M/M1KjAP3v2PAvKKAAVpJ/TOQ/FkI4V02T7qOY1kWs712LuG8jUr+T5Wovp0EBBbBixBifkf+4dKuk/3yOdsNNScl07c4QwgZJP7G43Tm1GOOM6gP3T3auvztediigAFaa303k7wkhXLGYFYcQ
```

Comment:
can we have a graphical demonstration of the model key idea?

Reply:
that's not necessary, just grab the code :)

Reply:
I think it would make this much more informative. a quick read through right now leaves me still not very clear about what exactly is happening here.

Reply:
me neither :), you do know the best thing we can do is simply use the same figure presented in their paper right?   if that's the case, I don't see presenting their figure in the notebook is adding anything valuable.